### PR TITLE
element asked for challenge 2

### PR DESCRIPTION
**Pull request by RozkeAdine**

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#projet_global{
+    width: 100%;
+
```
Comment (Sarawilong) [inline CSS/style.css:3]: useless line

File: index.html [hunk 2]
```diff
@@ -4,41 +4,92 @@
...
 
-           <article>
-        <h2>titre 2</h2>
-        <img src="https://media.istockphoto.com/photos/jardin-colombia-picture-id1385882999?k=20&m=1385882999&s=612x612&w=0&h=y5-y2n0V2hVqtG8zBWDn093mQYkQyBv8-r33EJ2vy4M=">
-    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+           <article id="projet_global">
+        <h2>Projet Réalisés</h2>
+            <div id="projet_1">
+                <H3>projet 1</H3>
```
Comment (Sarawilong) [inline index.html:22]: Majuscule

File: index.html [hunk 3]
```diff
@@ -4,41 +4,92 @@
...
+                <img src="https://media.istockphoto.com/photos/jardin-colombia-picture-id1385882999?k=20&m=1385882999&s=612x612&w=0&h=y5-y2n0V2hVqtG8zBWDn093mQYkQyBv8-r33EJ2vy4M=" height="100" widht="470" alt="chateau en foret1">
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <divc id="projet_2">
```
Comment (Sarawilong) [inline index.html:27]: divc?

File: index.html [hunk 4]
```diff
@@ -4,41 +4,92 @@
...
+                <H3>projet 2</H3>
+                <img src="https://media.istockphoto.com/photos/jardin-colombia-picture-id1385882999?k=20&m=1385882999&s=612x612&w=0&h=y5-y2n0V2hVqtG8zBWDn093mQYkQyBv8-r33EJ2vy4M=" height="100" widht="470" alt="chateau en foret1">
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </divc>
+            <div id="projet_3">
+                <H3>projet 3</H3>
```
Comment (Sarawilong) [inline index.html:34]: Majuscule

File: index.html [hunk 5]
```diff
@@ -4,41 +4,92 @@
...
+                <img src="https://media.istockphoto.com/photos/jardin-colombia-picture-id1385882999?k=20&m=1385882999&s=612x612&w=0&h=y5-y2n0V2hVqtG8zBWDn093mQYkQyBv8-r33EJ2vy4M=" height="100" widht="470" alt="chateau en foret1">
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
```
Comment (Sarawilong) [inline index.html:38]: cs

File: index.html [hunk 6]
```diff
@@ -4,41 +4,92 @@
...
             </article>
```
Comment (Sarawilong) [inline index.html:40]: nothing in there ?

File: index.html [hunk 7]
```diff
@@ -4,41 +4,92 @@
...
 
             <article>
-        <h2>titre 3</h2>
-        <img src="https://media.istockphoto.com/photos/mezzanego-picture-id642427908?k=20&m=642427908&s=612x612&w=0&h=uqdzGIanyrdMKbi1HVDtATy5D0wj1sG-4if_ghNNnFA=">
-    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+        <h2>Articles</h2>
+            <div class="articles">
+                <H3>article 1</H3>
```
Comment (Sarawilong) [inline index.html:53]: majuscule

File: index.html [hunk 8]
```diff
@@ -4,41 +4,92 @@
...
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <div class =articles>
+                <H3>article 1</H3>
+                <img src="https://media.istockphoto.com/photos/mezzanego-picture-id642427908?k=20&m=642427908&s=612x612&w=0&h=uqdzGIanyrdMKbi1HVDtATy5D0wj1sG-4if_ghNNnFA="  height="100" widht="470" alt="chateau en foret3">
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+
```
Comment (Sarawilong) [inline index.html:64]: useless line

File: index.html [hunk 9]
```diff
@@ -4,41 +4,92 @@
...
 
     </section>
 
     <footer>
+        <p>Contact</p>
+        <form action="/data_form.txt" method="post">
+            <ul>
+              <li>
+                <label for="name">NAME:</label><br>
+                <input type="text" value="">
```
Comment (Sarawilong) [inline index.html:75]: id de l'input where is it ?

File: index.html [hunk 10]
```diff
@@ -4,41 +4,92 @@
...
+              </li>
+              <li>
+                <label for="mail">E-MAIL:</label><br>
+                <input type="email" value="@">
```
Comment (Sarawilong) [inline index.html:79]: same here

File: index.html [hunk 11]
```diff
@@ -4,41 +4,92 @@
...
+              </li>
+              <li>
+                <label for="msg">MESSAGE:</label><br>
+                <textarea></textarea>
+                <li class="button">
+                  <button type="submit">envoyer</button>
```
Comment (Sarawilong) [inline index.html:85]: Majuscule

Comment (Sarawilong) [reply index.html:85]: cs

File: index.html [hunk 12]
```diff
@@ -4,41 +4,92 @@
...
+                </li>
+              </li>
+            </ul>
+        </form>
 
-        <p>nous contacter</p>
-        <img src="https://cdn.pixabay.com/photo/2017/08/16/00/59/panorama-2646143_960_720.jpg">
+        
+        <img src="https://cdn.pixabay.com/photo/2017/08/16/00/59/panorama-2646143_960_720.jpg"  height="200" widht="470" alt="image chateau en ville bors d'eau">
```
Comment (Sarawilong) [inline index.html:93]: useless line

**Review by Sarawilong — CHANGES_REQUESTED** [review]
Indentation à revoir

**Comment by RozkeAdine** [comment]
Hello
Sarawi,

Je me suis rendu compte de la majorité des fautes dont tu parles hier soir pendant que je travaillais encore sur le portfolio : j’aurais dû mieux vérifier mon code !!
Input : je ne savais pas qu’il lui fallait un id aussi.

Je te tiens au courant

Cordialement.

Le 24 mai 2022 à 14:18, Sarawilong ***@***.***> a écrit :

﻿

@Sarawilong requested changes on this pull request.

Indentation à revoir

________________________________

In CSS/style.css<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880424885>:

> @@ -0,0 +1,10 @@
+#projet_global{
+    width: 100%;
+


useless line

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880426760>:

>          </header>
     <section>

-           <article>
-        <h2>titre 2</h2>
-        
-    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+           <article id="projet_global">
+        <h2>Projet Réalisés</h2>
+            <div id="projet_1">
+                <H3>projet 1</H3>


Majuscule

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880427096>:

>          </header>
     <section>

-           <article>
-        <h2>titre 2</h2>
-        
-    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+           <article id="projet_global">
+        <h2>Projet Réalisés</h2>
+            <div id="projet_1">
+                <H3>projet 1</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <divc id="projet_2">


divc?

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880427347>:

> +           <article id="projet_global">
+        <h2>Projet Réalisés</h2>
+            <div id="projet_1">
+                <H3>projet 1</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <divc id="projet_2">
+                <H3>projet 2</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </divc>
+            <div id="projet_3">
+                <H3>projet 3</H3>


Majuscule

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880427532>:

> +                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <divc id="projet_2">
+                <H3>projet 2</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </divc>
+            <div id="projet_3">
+                <H3>projet 3</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>


cs

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880427742>:

> +                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <divc id="projet_2">
+                <H3>projet 2</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </divc>
+            <div id="projet_3">
+                <H3>projet 3</H3>
+                chateau en foret1
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
             </article>



nothing in there ?

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880428082>:

>              </article>

             <article>
-        <h2>titre 3</h2>
-        
-    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+        <h2>Articles</h2>
+            <div class="articles">
+                <H3>article 1</H3>


majuscule

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880428451>:

> -    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
-    et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+        <h2>Articles</h2>
+            <div class="articles">
+                <H3>article 1</H3>
+                chateau en foret3
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+            <div class =articles>
+                <H3>article 1</H3>
+                chateau en foret3
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis illo voluptate, molestias delectus quia,
+                et numquam repudiandae placeat quasi alias vel excepturi dignissimos tenetur dolor cum assumenda accusantium fugiat at.</p>
+            </div>
+


useless line

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880429579>:

>              </article>

     </section>

     <footer>
+        <p>Contact</p>
+        <form action="/data_form.txt" method="post">
+            <ul>
+              <li>
+                <label for="name">NAME:</label><br>
+                <input type="text" value="">


id de l'input where is it ?

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880429718>:

>              </article>

     </section>

     <footer>
+        <p>Contact</p>
+        <form action="/data_form.txt" method="post">
+            <ul>
+              <li>
+                <label for="name">NAME:</label><br>
+                <input type="text" value="">
+              </li>
+              <li>
+                <label for="mail">E-MAIL:</label><br>
+                <input type="email" value="@">


same here

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880429951>:

> +        <p>Contact</p>
+        <form action="/data_form.txt" method="post">
+            <ul>
+              <li>
+                <label for="name">NAME:</label><br>
+                <input type="text" value="">
+              </li>
+              <li>
+                <label for="mail">E-MAIL:</label><br>
+                <input type="email" value="@">
+              </li>
+              <li>
+                <label for="msg">MESSAGE:</label><br>
+                <textarea></textarea>
+                <li class="button">
+                  <button type="submit">envoyer</button>


Majuscule

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880430264>:

>
-        <p>nous contacter</p>
-        
+
+        image chateau en ville bors d'eau



useless line

________________________________

In index.html<https://github.com/RozkeAdine/DonkeySchool/pull/2#discussion_r880430146>:

> +        <p>Contact</p>
+        <form action="/data_form.txt" method="post">
+            <ul>
+              <li>
+                <label for="name">NAME:</label><br>
+                <input type="text" value="">
+              </li>
+              <li>
+                <label for="mail">E-MAIL:</label><br>
+                <input type="email" value="@">
+              </li>
+              <li>
+                <label for="msg">MESSAGE:</label><br>
+                <textarea></textarea>
+                <li class="button">
+                  <button type="submit">envoyer</button>


cs

**Pull request merged**